patterns/stories-podcast-hero.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Title: Stories Podcast Hero
+ * Slug: twentytwentyfive/stories-podcast-hero
+ * Categories: featured, hero
+ * Keywords: podcast, hero, stories
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:group {"metadata":{"categories":["featured","hero"],"patternName":"twentytwentyfive/stories-podcast-hero","name":"Stories Podcast Hero"},"align":"full","className":"has-accent-2-background-color has-background","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"}},"backgroundColor":"accent-2"},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull has-accent-2-background-color has-background" style="padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)"><!-- wp:columns {"align":"wide"} -->
+	<div class="wp-block-columns alignwide"><!-- wp:column {"width":"40%"} -->
+		<div class="wp-block-column" style="flex-basis:40%"><!-- wp:image {"width":"360px","height":"360px","sizeSlug":"large","linkDestination":"none"} -->
+		<figure class="wp-block-image size-large"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/podcast-hero.webp" alt="<?php echo esc_attr_x( 'Picture of a person', 'Podcast Hero Image', 'twentytwentyfive' ); ?>" width="360" height="360"/></figure>
```

Comment:
Why was 360px chosen here?
Does it work without any size set, but with an aspect ratio?
The design in Figma uses 485px  X 485px.

Reply:
hey @carolinan so I dont know where to get the source image from - I have checked but I dont have edit perms on the figmna so I took a screenshot as a placeholder. If I can be directed to where this image lives I can make a replacement for something the right size however it does work with 485 x 485 like the figma, I had it that was in the prev PR this was an error on my side

Reply:
The project readme file has now been updated with some more information and a new Google Drive folder with the images stored in it.
Here is a link to the image:
https://drive.google.com/file/d/17JaQF-w8wYMnCZfSeiTO03zdlK7Sec8t/view?usp=drive_link

And this is the actual page where the image is from:
https://www.rawpixel.com/image/14264288